wscript

```diff
@@ -482,6 +484,12 @@ def build(ctx):
 			use_extra.append('HDF5')
 			use_extra.append('lib/julea-kv')
 			use_extra.append('lib/julea-object')
+		elif client == 'hdf5-db':
+			use_extra.append('HDF5DB')
+			use_extra.append('SQLITE')
```

Comment:
Why does it need SQLite?

Reply:
It does not .. that is from an old version, where it was required.